library/cwm/src/lib/cwm/widget.rb

```diff
@@ -98,6 +98,8 @@ def self.widget_type=(type)
     #   @return [String] translated label text for the widget
 
     # @!method opt
+    #   Specifies options passed to widget. When {handle} method defined, then
```

Comment:
What about this? When {handle} method *is* defined, is expected to be immediatelly notified.

Or, at least, replace *immeditally* for *inmediately*.